### DISCUSSION TITLE
Archery 與Nexus 掃描漏洞弱點修補

### DISCUSSION THREAD

**Discussion by dearsiang** [hhyo/Archery#2774] [Q&A]
Hi 大大們

最近我們使用Nexus 對Archery 掃描漏洞，掃出來如下

Critical
MySQL Default Account Credentials
CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H
Recommendation: Either remove the affected accounts or change the associated password.

Oracle MySQL Server 5.7.x < 5.7.44 (October 2023 CPU)
CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H
Recommendation: Apply the appropriate patch according to the October 2023 Oracle Critical Patch Update advisory. Highly recommended to upgrade MySQL to 8.0.39 or above.

High
MySQL User-Defined Functions Multiple Vulnerabilities
CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:H/I:H/A:H
Recommendation: There is currently no known fix or patch to address these issues. Instead, make sure access to create user-defined functions is restricted.

nginx 1.1.x < 1.23.2 / 1.0.x < 1.22.1 Memory Disclosure
CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:H/I:H/A:H
Recommendation: Upgrade to nginx 1.26 or later.

MySQL Unpassworded Account Check
CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:L/I:L/A:L
Recommendation:Disable or set a password for the affected account.

MySQL Zero-length Scrambled String Crafted Packet Authentication Bypass
CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:L/I:L/A:L
Recommendation:Upgrade MySQL to the latest supported version.

nginx 0.6.x < 1.20.1 1-Byte Memory Overwrite RCE
CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:L/I:L/A:L
Recommendation:Upgrade to nginx 1.26 or later.

請問那裡可以看到這些漏洞的更新計畫？或是我們使用者可以手動做哪些調整修補這些漏洞呢？
![Uploading archery1.png…]()

**Comment by dearsiang** [comment]
<img width="1222" alt="archery2" src="https://github.com/user-attachments/assets/fa8834c9-659b-45c1-8b04-3113de097e56">
<img width="1387" alt="archery3" src="https://github.com/user-attachments/assets/ebab1fd3-c978-45e7-bf13-e3bd92f8deeb">
<img width="602" alt="archery4" src="https://github.com/user-attachments/assets/ee25a71f-e692-4699-8049-f3da7f7c6cf3">

**Comment by dearsiang** [comment]
<img width="1231" alt="archery1" src="https://github.com/user-attachments/assets/d29cea56-e3ae-4cf9-9103-512bfecf4a9f">

**Comment by nick2wang** [comment]
以上关于4000端口的mysql漏洞基本都是goinception默认关闭用户鉴权引起，参考：[goinception用户鉴权](https://hanchuanchuan.github.io/goInception/zh/safe.html)

**Reply by dearsiang** [reply]
請問Archery 有支援mysql 8.0.39嗎？

使用docker 方式安裝

**Reply by nick2wang** [reply]
支持的
